### PR TITLE
Hybridizing component

### PR DESCRIPTION
**Pull request by ta1188**
Updating component to hybrid status.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,12 +4,28 @@
...
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0"
+  },
+  "resolutions": {
+    "webcomponentsjs": "^1.0.0"
+  },
+  "variants": {
+   "1.x": {
+     "devDependencies": {
+       "iron-component-page": "PolymerElements/iron-component-page#^2.0.0",
```
Comment (dcravenus) [inline bower.json]: Shouldn't this be 1.0.0

File: bower.json [hunk 2]
```diff
@@ -4,12 +4,28 @@
   "version": "2.0.0",
   "main": "fs-memories-util-behavior.html",
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "polymer": "Polymer/polymer#1.9 - 2"
   },
   "devDependencies": {
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
```
Comment (dcravenus) [inline bower.json]: Shouldn't this be 2.0.0

**Comment by dcravenus** [comment]
It looks like you updated the 1.x dependencies but not the 2.x ones for some reason.

File: bower.json (hunk 1 above)
Comment (ta1188) [reply bower.json]: Seems to be that `2.0` is their Hybrid version for this one...
https://github.com/PolymerElements/iron-component-page/releases

File: bower.json (hunk 2 above)
Comment (ta1188) [reply bower.json]: Don't I need a range for "hybrid" abilities?

File: bower.json (hunk 1 above)
Comment (dcravenus) [reply bower.json]: If you are using the hybrid version of iron-component-page, then you should probably update that in the main devDependencies and remove the entry from the 1.x devDependencies

File: bower.json (hunk 2 above)
Comment (dcravenus) [reply bower.json]: If v2 of iron-component-page is hybrid then no you don't need a range

File: bower.json (hunk 1 above)
Comment (ta1188) [reply bower.json]: Haha, mis-matched the versions...fixed now

File: bower.json (hunk 2 above)
Comment (dcravenus) [reply bower.json]: You use the range if there is not hybrid version and you have to pick one or the other like Polymer itself.

**Pull request merged**